lib/pubspec.dart

```diff
@@ -3,7 +3,7 @@
 // ignore_for_file: public_member_api_docs, constant_identifier_names, avoid_classes_with_only_static_members
 
 mixin Pubspec {
-  static final buildDate = DateTime.utc(2024, 5, 7, 18, 3, 6);
+  static final buildDate = DateTime.utc(2024, 6, 25, 23, 41, 21);
```

Comment:
This seems like a weird starting value? Should it default to now/today or something instead?

Reply:
pubspec.dart is generated automatically when `flutter pub run pubspec_extract`